support/node-production-aws-sns/index.js

```diff
@@ -0,0 +1,62 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const AWS = require('aws-sdk');
+const nodeCron = require('node-cron');
+const settings = require('./settings.json')
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress, types) {
+    const provider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider, types });
+    await api.isReady;
+    return api;
+}
+
+async function publishToTopic(topicName, message) {
+    AWS.config.update({region: settings.region});
+    const sns = new AWS.SNS({apiVersion: settings.awsApiVersion});
+    const topics = await sns.listTopics().promise();
+    const topicArn = topics.Topics.find(t => t.TopicArn.includes(topicName)).TopicArn;
+
+    return sns.publish({
+        Message: message,
+        TopicArn: topicArn,
+    }).promise();
+}
+
+async function main() {
+    const api = await createPromiseApi(settings.nodeAddress);
+
+    let lastHeaderHex = settings.lastHeaderHex;
+    let counter = 0;
+    let sendingSns = false;
+    const job = nodeCron.schedule(settings.cronTime, () => {
+        console.log(new Date().toLocaleString());
```

Comment:
Should log what we're doing. `console.log(new Date().toISOString(), 'Checking for new block');`. Or use a proper logging library that will format with relevant context.